### PR TITLE
CR-1098780 Removed the message regarding the emconfig.json for sw_emu…

### PR DESCRIPTION
**Pull request by venkatp-xilinx**
CR-1098780 Removed the message regarding the emconfig.json for sw_emu in edge cases

### CODE REVIEW

**Comment by dayeh-xilinx** [comment]
retest this please. Build encounter a strange error "'build.bat' is not recognized as an internal or external command". Try to retest.

**Comment by abalasa** [comment]
retest this please

File: src/runtime_src/core/edge/sw_em/generic_pcie_hal2/halapi.cxx [hunk 1]
```diff
@@ -95,12 +95,12 @@ xclDeviceHandle xclOpen(unsigned deviceIndex, const char *logfileName, xclVerbos
...
   if (handle) {
     handle->xclOpen(logfileName);
-    if (bDefaultDevice)
+    /*if (bDefaultDevice)
     {
       std::string sDummyDeviceMsg = "INFO: [SW-EM 09-0] Unable to find emconfig.json. Using default device.";
       if (xclemulation::config::getInstance()->isInfosToBePrintedOnConsole())
         std::cout << sDummyDeviceMsg << std::endl;
-    }
+    }*/
```
Comment (stsoe) [inline src/runtime_src/core/edge/sw_em/generic_pcie_hal2/halapi.cxx:103]: Please delete dead code

**Review by stsoe — CHANGES_REQUESTED** [review]
Just one minor comment.  Else good.

**Pull request merged**